### ISSUE TITLE
Query about code

### ISSUE THREAD

**Issue by Gabant** [sched-ext/sched_ext#218]
Hi Team
Could you please check if the following code has issues? I'm not sure if I understood this code correctly.
In the new version of the kernel code, the address of idle is the largest. And it seems the purpose of this code and the meaning of the comment do not match.
```
static enum dispatch_to_local_dsq_ret dispatch_to_local_dsq(...)
{
		... ...
		/* if the destination CPU is idle, wake it up */
		if (dsp && p->sched_class > dst_rq->curr->sched_class)
			resched_curr(dst_rq);
}
```

**Comment by ByteLab-David** [comment]
Good catch! Thanks for looking diligently at the code.

**Comment by ByteLab-David** [comment]
Fixed by #219
